<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>506</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ETerminal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCalls">
    <property name="title">
     <string>Calls</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuCalls"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>150</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>300</width>
     <height>150</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QPushButton" name="rescan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>ReScan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonConnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="selectBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Select Serial Port</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="serialPortInfoListBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="parametersBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>451</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Select Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>BaudRate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="baudRateBox"/>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="parityBox"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="flowControlLabel">
        <property name="text">
         <string>Flow control:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Data bits:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Parity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="dataBitsBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Stop bits:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="stopBitsBox"/>
      </item>
      <item row="0" column="5">
       <widget class="QComboBox" name="flowControlBox"/>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="localEchoCheckBox">
        <property name="text">
         <string>Local echo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearScreen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startLog">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>StartLog</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send_file">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send File</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About program</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;onnect</string>
   </property>
   <property name="toolTip">
    <string>Connect to serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset>
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from serial port</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="icon">
    <iconset>
     <normaloff>:/images/settings.png</normaloff>:/images/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configure</string>
   </property>
   <property name="toolTip">
    <string>Configure serial port</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset>
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>C&amp;lear</string>
   </property>
   <property name="toolTip">
    <string>Clear data</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset>
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
